src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
```

Comment:
since you have a method `getSalaryInfo()` it might have sense to name this class differently

how about `SalaryInfoProvider`?